.golden/swiftBasicNewtypeJoinOptionalsSpec/golden

```diff
@@ -1,4 +1,4 @@
 struct Newtype {
     typealias NewtypeTag = Tagged<Newtype, Int?>
```

Comment:
What is this test testing? Will a single `Maybe` still generate this question mark, because it should not

Reply:
It looks like it does the right thing in your other PR - should we add a test for that?

Reply:
This is the nested Maybe case